src/components/molecules/Chatbox/components/ChatboxOptionsControls/ChatboxOptionsControls.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from "react";
+import { Dropdown, DropdownButton } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { ChatMessageOptions, ChatOption } from "types/chat";
+
+import { TextButton } from "components/atoms/TextButton";
+
+import "./ChatboxOptionsControls.scss";
+
+export interface ChatboxOptionsControlsProps {
+  activeOption?: ChatOption;
+  setActiveOption: React.Dispatch<React.SetStateAction<ChatOption | undefined>>;
+}
+
+export const ChatboxOptionsControls: React.FC<ChatboxOptionsControlsProps> = ({
+  activeOption,
+  setActiveOption,
+}) => {
+  const showPoll = activeOption === ChatMessageOptions.poll;
```

Comment:
this naming implies it's a function. `isPoll` is a better choice

Reply:
Or prefix with `should`: `shouldShowPoll`

Reply:
fixed in f2fa24aadcae7547799e463bd593e8a7a2cf05b5

Reply:
@yurii-lubynets What is the fix that was made in that commit? I can't see it searching for any of the keywords mentioned above?

Reply:
@0xdevalias kindly apologize for that. It's was not pushed. Fixed here7ad21bd95185504846fc1206d006471e7a093219

Reply:
No worries, thanks for that. Looks good to me (though will leave this for @mike-lvov to resolve as he first opened it)